packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -412,7 +416,7 @@ export class SpecBuilder {
 
   watchSends(
     targets: { affects?: Query[]; removed?: Query[] },
-    ...docs: Document[]
+    ...docs: MaybeDocument[]
```

Comment:
Hrm...  I'm not sure this makes sense. Watch doesn't have a way to send "deleted" documents via the protos, does it?

Reply:
It does when it knows when a document was deleted: https://github.com/firebase/firebase-js-sdk/blob/fa5d157751ae64524691150f6bb512b94d86b6f6/packages/firestore/src/remote/watch_change.ts#L63